jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c

```diff
@@ -131,6 +145,38 @@ ecma_builtin_error_prototype_object_to_string (ecma_value_t this_arg) /**< this
   return ecma_make_string_value (ecma_stringbuilder_finalize (&builder));
 } /* ecma_builtin_error_prototype_object_to_string */
 
+/**
+  * Dispatcher of the Generator built-in's routines
+  *
+  * @return ecma value
+  *         Returned value must be freed with ecma_free_value.
+  */
+ecma_value_t
+ecma_builtin_error_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine
+                                                                                      *   identifier */
+                                                         ecma_value_t this_arg, /**< 'this' argument value */
+                                                         const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                                 *   passed to
```

Comment:
No need to split a comment to 3 lines when there is enough space. Try to reduce these.